src/netex-convertor/netexGenerator.ts

```diff
@@ -271,15 +280,110 @@ const netexGenerator = (
     const updatePriceFareFrame = (priceFareFrame: NetexObject): NetexObject => {
         const priceFareFrameToUpdate = { ...priceFareFrame };
 
-        priceFareFrameToUpdate.id = `epd:UK:${coreData.operatorIdentifier}:FareFrame_UK_PI_FARE_PRODUCT:${coreData.placeholderGroupOfProductsName}@pass:op`;
+        const ticketIdentifier: string = isPointToPointTicket(ticket)
+            ? coreData.lineIdName
+            : coreData.placeholderGroupOfProductsName;
+
+        priceFareFrameToUpdate.id = `epd:UK:${coreData.operatorIdentifier}:FareFrame_UK_PI_FARE_PRODUCT:${ticketIdentifier}@pass:op`;
```

Comment:
The old pointToPointNetexGenerator doesn't seem to have the word 'pass' at the end of the `priceFareFrame.id`. Not sure if having this in here would break things for pointToPoint tickets?